src/mol-plugin-ui/controls/icons.tsx

```diff
@@ -133,6 +133,8 @@ const _GetApp = <svg width='24px' height='24px' viewBox='0 0 24 24'><path d='M19
 export function GetAppSvg() { return _GetApp; }
 const _HelpOutline = <svg width='24px' height='24px' viewBox='0 0 24 24'><path d='M11 18h2v-2h-2v2zm1-16C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5 0-2.21-1.79-4-4-4z' /></svg>;
 export function HelpOutlineSvg() { return _HelpOutline; }
+const _Tour = <svg width='24px' height='24px' viewBox='0 0 24 24'><path d='M21 4H7V2H5v20h2v-8h14l-2-5zm-6 5c0 1.1-.9 2-2 2s-2-.9-2-2 .9-2 2-2 2 .9 2 2' /></svg>;
```

Comment:
just to be sure, are the icons you added from https://material-ui.com/components/material-icons/? if not please put them in a different section and mention their license

Reply:
good point I will check